nats/vaz.yml

```diff
@@ -0,0 +1,50 @@
+type: json
+data:
+  url: /varz
+  server: ${NATS_HOSTPORT}
+
+observation:
+  - name: varz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: msgs.in
+        source: in_msgs
+        type: long_gauge
+        label: incoming messages
+        description: The amount of incoming messages
+
+      - name: bytes.in
+        source: in_bytes
+        type: long_gauge
+        label: incoming bytes
+        description: The amount of incoming bytes
```

Comment:
the unit is missing. I assume it is `bytes`. Same for outgoing bytes

Reply:
Done